.github/actions/docker-build-and-push/action.yaml

```diff
@@ -68,143 +68,143 @@ runs:
         flavor: |
           latest=false
 
     - name: Docker meta for autoware:universe-sensing-perception
       id: meta-universe-sensing-perception
       uses: docker/metadata-action@v5
       with:
-        images: ghcr.io/${{ github.repository_owner }}/${{ inputs.bake-target }}
+        images: ghcr.io/${{ github.repository_owner }}/${{ inputs.target-image }}
         tags: |
           type=raw,value=universe-sensing-perception-${{ inputs.platform }}
           type=raw,value=universe-sensing-perception-${{ steps.date.outputs.date }}-${{ inputs.platform }}
-        bake-target: docker-metadata-action-universe-sensing-perception
+        target-image: docker-metadata-action-universe-sensing-perception
         flavor: |
           latest=false
 
     - name: Docker meta for autoware:universe-localization-mapping-devel
       id: meta-universe-localization-mapping-devel
       uses: docker/metadata-action@v5
       with:
-        images: ghcr.io/${{ github.repository_owner }}/${{ inputs.bake-target }}
+        images: ghcr.io/${{ github.repository_owner }}/${{ inputs.target-image }}
         tags: |
           type=raw,value=universe-localization-mapping-devel-${{ inputs.platform }}
           type=raw,value=universe-localization-mapping-devel-${{ steps.date.outputs.date }}-${{ inputs.platform }}
-        bake-target: docker-metadata-action-universe-localization-mapping-devel
+        target-image: docker-metadata-action-universe-localization-mapping-devel
         flavor: |
           latest=false
 
     - name: Docker meta for autoware:universe-localization-mapping
       id: meta-universe-localization-mapping
       uses: docker/metadata-action@v5
       with:
-        images: ghcr.io/${{ github.repository_owner }}/${{ inputs.bake-target }}
+        images: ghcr.io/${{ github.repository_owner }}/${{ inputs.target-image }}
         tags: |
           type=raw,value=universe-localization-mapping-${{ inputs.platform }}
           type=raw,value=universe-localization-mapping-${{ steps.date.outputs.date }}-${{ inputs.platform }}
-        bake-target: docker-metadata-action-universe-localization-mapping
+        target-image: docker-metadata-action-universe-localization-mapping
         flavor: |
           latest=false
 
     - name: Docker meta for autoware:universe-planning-control-devel
       id: meta-universe-planning-control-devel
       uses: docker/metadata-action@v5
       with:
-        images: ghcr.io/${{ github.repository_owner }}/${{ inputs.bake-target }}
+        images: ghcr.io/${{ github.repository_owner }}/${{ inputs.target-image }}
         tags: |
           type=raw,value=universe-planning-control-devel-${{ inputs.platform }}
           type=raw,value=universe-planning-control-devel-${{ steps.date.outputs.date }}-${{ inputs.platform }}
-        bake-target: docker-metadata-action-universe-planning-control-devel
+        target-image: docker-metadata-action-universe-planning-control-devel
```

Comment:
https://github.com/docker/metadata-action?tab=readme-ov-file#inputs

The docker/metadata-action@v5 doesn't accept `target-image` parameter.

Reply:
Good catch. Thank you. https://github.com/autowarefoundation/autoware/pull/5413/commits/9c1feb04214c5054036e904e0a16738536e8ed78